src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1262,11 +1290,19 @@ class AbstractUtxoCoin extends BaseCoin {
 
       const queries = [];
 
-      _.forEach(AbstractUtxoCoin.AddressTypes, function(addressType) {
+      _.forEach(Object.keys(Codes.UnspentTypeTcomb.meta.map), function(addressType) {
         // If we aren't ignoring the address type, we derive the public key and construct the query for the main and change indices
```

Comment:
Update comment to use "external" and "internal" terminology.

Reply:
Done